PredictiveMethods/CaseOutcome/PredictiveMonitor.py

```diff
@@ -65,4 +65,4 @@ def test(self, dt_test):
         for label_col_idx, label in enumerate(pred_model.cls.classes_):
             if label == self.pos_label:
                 # output probability of a positive class
-                print(np.round(predictions_proba[0, label_col_idx], decimals=3))
+                return np.round(predictions_proba[0, label_col_idx], decimals=3)
```

Comment:
you can do both: print to stdout and return I think. The ideal case would be to preserve the possibility to run the project in dev without kafka and in production with it. Otherwise, it might be difficult to work with it in local, unless we make really good dockerized microservices architecture. What do think?